draft-ietf-core-href.md

```diff
@@ -425,36 +425,41 @@ represented by using the `discard` alternative instead.
 The rules `scheme`, `authority`, `path`, `query`, `fragment`
 correspond to the (sub-)components of a CRI, as described in
 {{constraints}}, with the addition of the `discard` section.
-The `discard` section can be used when neither a scheme nor an
-authority is present.
-It then expresses path
-prefixes such as "/",
-"./", "../", "../../", etc.
+
+### The `discard` Section
+
+The `discard` section can be used in a CRI reference when neither a
+scheme nor an authority is present.
+It then expresses the operations performed on a base CRI by CRI references that
+are equivalent to URI references with relative paths and path prefixes such as "/", "./", "../", "../../", etc.
+E.g., a simple URI reference "foo" specifies to remove one leading segment
+from the base URI's path, which is represented in the equivalent CRI
+reference discard section as the value `1`; similarly "../foo", two segments,
+`2`; and "/foo", all segments, represented in the `discard` section as the value `true`.
```

Comment:
```suggestion
E.g., a simple URI reference "foo" can specify to remove one leading segment
from the base URI's path by setting discard to `1` in the equivalent CRI
reference; similarly "../foo", two segments,
`2`; and "/foo", all segments, is represented in the `discard` section as the value `true`.
```

Reply:
Er, no: the URI reference "foo"  *does* specify to remove one segment.
Sorry for the parameterized semicolon list; please see CMOS what that means :-)

Reply:
I redundantify-ized the semicolon list.